pkg/backporter/backporter_test.go

```diff
@@ -0,0 +1,217 @@
+package backporter
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+
+	"k8s.io/test-infra/prow/bugzilla"
+	"k8s.io/utils/diff"
+)
+
+var (
+	landingPage    = fmt.Sprintf(htmlPageStart, "Home") + htmlPageEnd
+	clonesHTMLPage = fmt.Sprintf(htmlPageStart, "Clones") + clonesHTMLSubPage + htmlPageEnd
+	errorPage      = fmt.Sprintf(htmlPageStart, "Not Found") + errorSubPage + htmlPageEnd
+)
+
+const clonesHTMLSubPage = `
+	<h2 id="bugid"> <a href = "#bugid"> 1: Sample bug to test implementation of clones handler </a> | Status:  </h2>
+	<p> Target Release: [] </p>
+	
+		<p> No linked PRs! </p>
+	
+	
+		<p> Cloned From: <a href = "/getclones?ID=0"> Bug 0: Sample bug to test implementation of clones handler</a> | Status: 
+	
+	<h4 id="clones"> <a href ="#clones"> Clones</a> </h4>
+	<table class="table">
+		<thead>
+			<tr>
+				<th title="Targeted version to release fix" class="info">Target Release</th>
+				<th title="ID of the cloned bug" class="info">Bug ID</th>
+				<th title="Status of the cloned bug" class="info">Status</th>
+				<th title="PR associated with this bug" class="info">PR</th>
+			</tr>
+		</thead>
+		<tbody>
+		
+			
+				<tr>
+					<td style="vertical-align: middle;">[]</td>
+					<td style="vertical-align: middle;"><a href = "/getclones?ID=0">0</a></td>
+					<td style="vertical-align: middle;"></td>
+					<td style="vertical-align: middle;">
+						
+					</td>
+				</tr>
+			
+		
+		</tbody>
+	</table>`
+
+const errorSubPage = `Bug#1000 not found`
+
+func unwrapper(h HandlerFuncWithErrorReturn, t *testing.T) http.HandlerFunc {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		err := h(w, r)
+		if err != nil {
+			// Error needs to be checked for the linter, but cant be ignored since the tescases check for error states
```

Comment:
Sorry I don't understand? What keeps you from doing `_ = h(w, r)` if you do not care about the handler error?

Reply:
Nothing actually, I can do that.